war/src/main/js/widgets/variables.less

```diff
@@ -1,16 +1,8 @@
 :root {
-  --medium-translucent: #bfffffff;
+  --medium-translucent: rgba(255, 255, 255, 0.75);
 
-  --pale-blue-trans: #7f78b4dc;
```

Comment:
Will the removal of all of this not cause issues with some other part of the code?

Reply:
```
$ rg '\-\-pale-blue-trans'
war/src/main/js/widgets/variables.less
4:  --pale-blue-trans: #7f78b4dc;
```
it's unused